core/src/main/java/org/commonjava/indy/core/change/StoreEnablementManager.java

```diff
@@ -76,7 +80,7 @@ public void onStoreEnablementChange( @Observes ArtifactStoreEnablementEvent even
                     int timeout = Integer.parseInt( toStr );
                     try
                     {
-                        setReEnablementTimeout( store.getKey(), timeout );
+                        setReEnablementTimeout( store.getKey() );
```

Comment:
After a checking, I did not find a metadata setting in store for this "DISABLE_TIMEOUT", so I'm guessing this metadata is not used in indy. So I'll remove that condition.